infra/docker/jobservice/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.7-slim-buster
+
+USER root
+WORKDIR /feast
+
+COPY sdk/python sdk/python
+COPY Makefile Makefile
+COPY protos protos
+
+# Install make
+RUN apt-get update && apt-get -y install make git
+
+# Install Python dependencies
+RUN make compile-protos-python
+
+# Install Feast SDK
+COPY .git .git
```

Comment:
Is there a reason why we copy git into the docker file?

Reply:
Looks like git is used by sdk/python/setup.py here https://github.com/feast-dev/feast/blob/master/sdk/python/setup.py#L52. If I remove this line docker build step errors out.

Also I copied this mostly from the jupyter dockerfile, which contains the similar build step. We could improve docker build files by removing the dependency on git (seems like it's only used to copy the readme file from the git repo root), but I think that's probably better handled by another PR.